docs/admin/system-information.rst

```diff
@@ -955,6 +955,9 @@ Table schema
     * - ``partition_ident``
       - The partition ident of a partitioned table. Empty for non-partitioned tables.
       - ``TEXT``
+    * - ``index_uuid``
+      - Internal id for the index the shard is part of.
```

Comment:
I'd like to avoid exposing this as `index_` as this is very implementation detail and may also confuse other uses of this (lucene index, column index). Or any better suggestion?
```suggestion
    * - ``storage_uuid``
      - Internal id for the storage entity (table or partition) the shard is part of.
```

Reply:
Sounds good!